pages/_app.tsx

```diff
@@ -21,5 +40,9 @@ const MyApp = ({ Component, pageProps }: AppProps): React.ReactElement => {
 
 MyApp.getInitialProps = async (appContext: AppContextType<Router>) => ({ ...await App.getInitialProps(appContext) })
 
-
 export default appWithTranslation(MyApp);
+
+MyApp.propTypes = {
+  Component: PropTypes.elementType.isRequired,
+  pageProps: PropTypes.object.isRequired,
+};
```

Comment:
Typescript now complains that the type of MyApp doesn't fit. Maybe because of the additional propTypes. But we can also fix that later...

Reply:
Getting rid of propTypes and reverting line 15 to what it was before (i.e. using React.ReactElement instead of React.ReactNode fixes this